src/commands/utility/info/category.js

```diff
@@ -0,0 +1,36 @@
+const { Command, SwitchbladeEmbed, CommandError } = require('../../../')
+const moment = require('moment')
+
+module.exports = class InfoCategory extends Command {
+  constructor (client) {
+    super({
+      name: 'category',
+      parent: 'info',
+      requirements: { guildOnly: true },
+      parameters: [{
+        type: 'channel',
+        acceptCategory: true,
+        required: true,
+        full: true,
+        missingError: 'commands:info.subcommands.voice.missingChannelName'
```

Comment:
Wrong string. Should be `That category doesn't exist or I couldn't find it.`

Reply:
Category is a channel

Reply:
Yeah, I know, but it's just to look better, yk